src/app/frontend/common/components/label/label_controller.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @final
+ */
+export default class LabelController {
+  /**
+   * Constructs label controller.
+   * @ngInject
+   */
+  constructor() {
+    /** @export {!Array<Object.<string,string>>} Initialized from the scope and transformed to
+     *  correct format */
+    this.labels = this.transformLabels_();
+  }
+
+  /**
+   * Transforms labels object into array of simple key, value label objects. It is required
+   * in order to use md-chips for label display.
+   * @return {!Array<Object.<string,string>>}
+   * @private
+   */
+  transformLabels_() {
```

Comment:
The reason why I'm doing this is because md-chips needs array and what we get from backend is 1 object with lots of properties like `{app: 'name', version: 'v1'}`. I'm transforming object into array and md-chips iterates over this array. 

It's based on this demo:
https://material.angularjs.org/latest/demo/chips